mycroft/audio/audioservice.py

```diff
@@ -105,25 +131,54 @@ def load_services(config, bus, path=None):
         except Exception as e:
             LOG.error('Failed to import module ' + descriptor['name'] + '\n' +
                       repr(e))
-            continue
-
-        if (hasattr(service_module, 'autodetect') and
-                callable(service_module.autodetect)):
-            try:
-                s = service_module.autodetect(config, bus)
-                service += s
-            except Exception as e:
-                LOG.error('Failed to autodetect. ' + repr(e))
-        if hasattr(service_module, 'load_service'):
-            try:
-                s = service_module.load_service(config, bus)
+        else:
+            s = setup_service(service_module, config, bus)
+            if s:
                 service += s
-            except Exception as e:
-                LOG.error('Failed to load service. ' + repr(e))
 
     return service
 
 
+def load_plugins(config, bus):
```

Comment:
could this be moved to utils.plugins?  `load_plugin()` is there, why not this?

Reply:
This loads plugins slightly differently from the TTS and STT since the audioservices are complete modules and can be loaded in a couple of different ways. This uses the mechanics defined in the util library and adapts it according to the needs of this specific service.

Adding a `load_audioservice_plugin()` to the general `mycroft.util.plugins` does to me seem a bit weird.